tests/payment/models/Payment - Credit card - Expired.model.js

```diff
@@ -0,0 +1,64 @@
+import { setup, visitAndClearCookies } from '../../../utils'
+import {
+  fillEmail,
+  getEmailWithSomeExpiredCard,
+  confirmSecondPurchase,
+  getEmailWithAllExpiredCards,
+} from '../../../utils/profile-actions'
+import {
+  selectCreditCardGroup,
+  queryIframe,
+  getIframeBody,
+} from '../../../utils/payment-actions'
+
+export default function test(account) {
+  describe(`Payment - Credit Card  - Expired - ${account}`, () => {
+    before(() => {
```

Comment:
I think this should be a `beforeEach` to keep consistency between tests.

The reason why all the other tests use `before` is because they have only a single `it` per test plan, so it might be a little bit misleading.

Reply:
Makes sense.